sklearn/linear_model/logistic.py

```diff
@@ -1789,3 +1790,37 @@ def fit(self, X, y, sample_weight=None):
 
         self.C_ = np.asarray(self.C_)
         return self
+
+    def score(self, X, y, sample_weight=None):
+        """Returns the score using the `scoring` option on the given
+        test data and labels.
+
+        Parameters
+        ----------
+        X : array-like, shape = (n_samples, n_features)
+            Test samples.
+
+        y : array-like, shape = (n_samples) or (n_samples, n_outputs)
```

Comment:
Do we support ``(n_samples, n_outputs)``? Seems that in ``fit``, we only support ``(n_samples,)``